composer.json

```diff
@@ -10,6 +10,7 @@
     ],
     "require": {
         "php": "^7",
+        "doctrine/persistence": "^1.3 || ^2 || ^3",
```

Comment:
Ik heb deze nu toegevoegd aan de require. Ik kan ook een "conflict" toevoegen met doctrine/persistence <1.3.0. Dan voorkomen we onderhoud als Doctrine 4, 5 of 6 uit komt 😄 
Officieel moet deze dependency er natuurlijk wel in. Maar Doctrine bundle, orm, collections, cache etc. worden ook gebruikt dus zouden er ook in moeten 🤔

Reply:
Composer gaat efficienter door een `range` check dan 3 separate versies die los van elkaar gecheckt moeten worden, mogelijk zou `^1.3 - ^3` ook werken

Reply:
> Could not parse version constraint ^1.3 - ^3: Invalid version string "^1.3 - ^3"